bindings/rust/build.rs

```diff
@@ -11,8 +11,22 @@ use std::env;
 use std::path::PathBuf;
 
 fn main() {
-    // This is the root directory.
-    let src = "../../";
```

Comment:
For some reason `cargo package` copies over symlinks to files with an updated path (that's why this works on evmc, where we only need the header which is symlinked), but symlinking the root does not work (e.g.` bindings/rust/fizzy -> ../../`).